src/Template/Element/sentences/translation.ctp

```diff
@@ -6,12 +6,26 @@ $sentenceBaseUrl = $this->Url->build([
     'controller' => 'sentences',
     'action' => 'show',
 ]);
+$css = CurrentUser::isTrusted() ? 'trusted-user' : '';
 ?>
 <div ng-repeat="translation in <?= $translations ?>"
-     class="translation" ng-class="{'not-reliable' : translation.correctness === -1}">
+     class="translation <?= $css ?>" ng-class="{'not-reliable' : translation.correctness === -1, 'expanded': vm.isMenuExpanded}">
     
     <div layout="row" layout-align="start center" flex>
-    <md-icon class="chevron">chevron_right</md-icon>
+    <?php if (CurrentUser::isTrusted()) { ?>
+        <md-button class="md-icon-button" ng-if="vm.isMenuExpanded && translation.isDirect" ng-click="vm.saveLink('delete', translation)">
+            <md-icon md-svg-src="/img/link_off.svg"></md-icon>
+            <md-tooltip><?= __('Unlink this translation.') ?></md-tooltip>
+        </md-button>
+        <md-button class="md-icon-button" ng-if="vm.isMenuExpanded  && !translation.isDirect" ng-click="vm.saveLink('add', translation)">
+            <md-icon>link</md-icon>
+            <md-tooltip><?= __('Make into direct translation.') ?></md-tooltip>
+        </md-button>
+        <md-icon class="chevron" ng-if="!vm.isMenuExpanded">chevron_right</md-icon>
+    <?php } else { ?>
+        <md-icon class="chevron">chevron_right</md-icon>
+    <?php } ?>
```

Comment:
I think it would be cleaner to have this `if (CurrentUser::isTrusted())` branch coded as an AngularJS `ng-if`. Maybe you could retrieve the value as a new canLink permission from the menu (we already have canEdit, canTranscribe…).